tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraCalendarEventAttendanceGetMethodContract.scala

```diff
@@ -228,8 +229,272 @@ trait LinagoraCalendarEventAttendanceGetMethodContract {
+      .map(_ + "")
+      .toArray
+    val blobIdsJson: String = Json.stringify(Json.arr(blobIds)).replace("[[", "[").replace("]]", "]")
+    val request: String =
+      s"""{
+         |  "using": [
+         |    "urn:ietf:params:jmap:core",
+         |    "com:linagora:params:calendar:event"],
+         |  "methodCalls": [[
+         |    "CalendarEventAttendance/get",
+         |    {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "blobIds": $blobIdsJson
+         |    },
+         |    "c1"]]
+         |}""".stripMargin
+
+    val response: String =
+      `given`
+        .body(request)
+      .when
+        .post
+      .`then`
+        .statusCode(SC_OK)
+        .contentType(JSON)
+        .extract
+        .body
+        .asString
+
+    assertThatJson(response)
+      .inPath("methodResponses[0]")
+      .isEqualTo(
+        s"""[
+           |    "error",
+           |    {
+           |        "type": "requestTooLarge",
+           |        "description": "The number of ids requested by the client exceeds the maximum number the server is willing to process in a single method call"
+           |    },
+           |    "c1"
+           |]""".stripMargin)
+  }
+
+  @Test
+  def shouldFailWhenWrongAccountId(): Unit = {
+    val request: String =
+      s"""{
+         |  "using": [
+         |    "urn:ietf:params:jmap:core",
+         |    "com:linagora:params:calendar:event"],
+         |  "methodCalls": [[
+         |    "CalendarEventAttendance/get",
+         |    {
+         |      "accountId": "unknownAccountId",
+         |      "blobIds": [ "0f9f65ab-dc7b-4146-850f-6e4881093965" ]
+         |    },
+         |    "c1"]]
+         |}""".stripMargin
+
+    val response =
+      `given`
+        .body(request)
+      .when
+        .post
+      .`then`
+        .statusCode(SC_OK)
+        .contentType(JSON)
+        .extract
+        .body
+        .asString
+
+    assertThatJson(response)
+      .inPath("methodResponses[0]")
+      .isEqualTo(
+        s"""[
+           |    "error",
+           |    {
+           |        "type": "accountNotFound"
+           |    },
+           |    "c1"
+           |]""".stripMargin)
+  }
+
+  @Disabled("A system session is used to fetch message flags from user inbox which does not seem to handle user mailbox isolation.")
+  @Test
+  def shouldNotFoundWhenDoesNotHavePermission(server: GuiceJamesServer): Unit = {
+    val blobId: String =
+      sendInvitationEmailToBobAndGetIcsBlobIds(server, "emailWithAliceInviteBobIcsAttachment.eml", icsPartId = "3")
+
+    val request: String =
+      s"""{
+         |  "using": [
+         |    "urn:ietf:params:jmap:core",
+         |    "com:linagora:params:calendar:event"],
+         |  "methodCalls": [[
+         |    "CalendarEventAttendance/get",
+         |    {
+         |      "accountId": "$ANDRE_ACCOUNT_ID",
+         |      "blobIds": [ "$blobId" ]
+         |    },
+         |    "c1"]]
+         |}""".stripMargin
```

Comment:
Tell me more about this please.

What is the actual behaviour?
Can we be sure Bob can do `CalendarEventAttendance/get` calls if, and only of the mailbox containing the blobs had been delegated to him? (Otherwise it is a -minor- security vulnerability)

Reply:
The problem boils down to `StandaloneEventAttendanceRepository#getFlags`.

When logging the returned message, it is actually returned even when the session user does not own that message.

Reply:
It is a false alarm, I probably misread the output of one test case to the permission one.